packages/components/src/components/hds/modal/body.ts

```diff
@@ -6,9 +6,16 @@
 import Component from '@glimmer/component';
 import { deprecate } from '@ember/debug';
 
-export default class HdsModalBodyComponent extends Component {
-  constructor() {
-    super(...arguments);
+export interface HdsModalBodySignature {
+  Blocks: {
+    default: [];
+  };
+  Element: HTMLDivElement;
+}
+
+export default class HdsModalBodyComponent extends Component<HdsModalBodySignature> {
+  constructor(owner: unknown) {
+    super(owner, {});
```

Comment:
Note: `super(arguments)` would get the `prefer-rest-params` ESLint error when I converted the file to TS (https://eslint.org/docs/latest/rules/prefer-rest-params)

To resolve, I ended up just typing the constructor arguments and being more specific about what is passed to `super()`. Since this component doesn't have any args, I just pass an empty object.